watertap/unit_models/tests/test_osmotically_assisted_reverse_osmosis_1D.py

```diff
@@ -87,7 +86,7 @@ def test_config():
     assert m.fs.unit.config.pressure_change_type == PressureChangeType.fixed_per_stage
     assert m.fs.unit.feed_side._flow_direction == FlowDirection.forward
     assert m.fs.unit.permeate_side._flow_direction == FlowDirection.backward
-    assert m.fs.unit.config.friction_factor == FrictionFactor.flat_sheet
+    assert m.fs.unit.config.module_type == ModuleType.flat_sheet
```

Comment:
@zacharybinger I suspect the same issue with flux calculation might also affect OARO 1D, though it's not currently evident from the OARO 1D tests since they don't cover Spiral wound scenarios. I suggest verifying that OARO Spiral wound operates as intended and implementing any required adjustments

Reply:
Fixed in the meeting, validated the results in the following figure, and added a OARO spiral-wound test
![membrane_width_fix_oaro](https://github.com/watertap-org/watertap/assets/23547346/966ba476-cdea-401d-a448-b66b80e1c0d4)

Reply:
Fixed in meeting, validated with the following figure, and added an OARO spiral-wound test
![membrane_width_fix_oaro](https://github.com/watertap-org/watertap/assets/23547346/05d62363-e720-49d5-a8d8-ffd384ef0636)

Reply:
Is the figure above for OARO? Did you also check the permeate channel pressure drop and osmotic pressure? I don't expect any dramatic findings but just thought I'd ask.

Reply:
Yeah permeate side checked out